zerver/webhooks/taiga/tests.py

```diff
@@ -11,225 +11,225 @@ def setUp(self) -> None:
         self.send_and_test_stream_message("issue_changed_severity", u'subject', message)
 
     def test_taiga_issue_changed_priority(self) -> None:
-        message = u':rocket: TomaszKolek changed priority of issue **New issue** from Normal to Low.'
+        message = u' TomaszKolek changed priority of issue **New issue** from Normal to Low.'
         self.send_and_test_stream_message("issue_changed_priority", u'subject', message)
 
     def test_taiga_issue_changed_comment_added(self) -> None:
-        message = u':thought_balloon: TomaszKolek commented on issue **New issue**.'
+        message = u' TomaszKolek commented on issue **New issue**.'
         self.send_and_test_stream_message("issue_changed_comment_added", u'subject', message)
 
     def test_taiga_epic_created(self) -> None:
-        message = u':package: Eeshan Garg created epic **Zulip is awesome!**'
+        message = u' Eeshan Garg created epic **Zulip is awesome!**'
         self.send_and_test_stream_message("epic_created", u'subject', message)
 
     def test_taiga_epic_changed_assigned(self) -> None:
-        message = u':busts_in_silhouette: Eeshan Garg assigned epic **Zulip is awesome!** to Eeshan Garg.'
+        message = u' Eeshan Garg assigned epic **Zulip is awesome!** to Eeshan Garg.'
         self.send_and_test_stream_message("epic_changed_assigned", u'subject', message)
 
     def test_taiga_epic_changed_unassigned(self) -> None:
-        message = u':busts_in_silhouette: Eeshan Garg unassigned epic **Zulip is awesome!**'
+        message = u' Eeshan Garg unassigned epic **Zulip is awesome!**'
         self.send_and_test_stream_message("epic_changed_unassigned", u'subject', message)
 
     def test_taiga_epic_changed_reassigned(self) -> None:
-        message = u':busts_in_silhouette: Eeshan Garg reassigned epic **Zulip is awesome!** from Eeshan Garg to Angela Johnson.'
+        message = u' Eeshan Garg reassigned epic **Zulip is awesome!** from Eeshan Garg to Angela Johnson.'
         self.send_and_test_stream_message("epic_changed_reassigned", u'subject', message)
 
     def test_taiga_epic_changed_blocked(self) -> None:
-        message = u':lock: Eeshan Garg blocked epic **Zulip is awesome!**'
+        message = u' Eeshan Garg blocked epic **Zulip is awesome!**'
         self.send_and_test_stream_message("epic_changed_blocked", u'subject', message)
 
     def test_taiga_epic_changed_unblocked(self) -> None:
-        message = u':unlock: Eeshan Garg unblocked epic **Zulip is awesome!**'
+        message = u' Eeshan Garg unblocked epic **Zulip is awesome!**'
         self.send_and_test_stream_message("epic_changed_unblocked", u'subject', message)
 
     def test_taiga_epic_changed_status(self) -> None:
-        message = u':chart_increasing: Eeshan Garg changed status of epic **Zulip is awesome!** from New to In progress.'
+        message = u' Eeshan Garg changed status of epic **Zulip is awesome!** from New to In progress.'
```

Comment:
There is an unnecessary space character at the start of every notification's text. I'd also remove that. :)